server.Dockerfile

```diff
@@ -14,21 +14,19 @@ RUN (cd ./temporal && go mod download all)
 COPY ./tctl/go.mod ./tctl/go.sum ./tctl/
 RUN (cd ./tctl && go mod download all)
 
-# cache Temporal CLI packages as a docker layer
-COPY ./cli/go.mod ./cli/go.sum ./cli/
-RUN (cd ./cli && go mod download all)
+# install Temporal CLI
+RUN sh -c "$(curl -sSf https://temporal.download/cli.sh)" -- --dir ./cli && \
```

Comment:
Does the custom dir change how cli config is looked up?

Reply:
no, CLI config is always under User home dir https://github.com/temporalio/tctl-kit/blob/577f95d16fa0c0cd1622931f7ce6758c31d13ab8/pkg/config/config.go#L223